src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/UpdateRetentionData.sql

```diff
@@ -93,7 +93,7 @@ BEGIN TRANSACTION
 						Set @UpdateColsList = ''
 
 					Set @FromClause  = ' FROM ' + @DataSetSchema + '.' + @DataSetTable +' ST ' + ' JOIN ' + @RefDataSetSchema + '.' + @RefDataSetTable + ' PT '				
-
+					-- Set @JOINClause  = ' ON PT.' + @DatasetJOINColumn + ' = ST.' + @RefColumn				
 					Set @JOINClause  = ' ON PT.' + @RefColumn + ' = ST.' + @DataSetJOINColumn				
```

Comment:
why is this changed, will that not have an effect on previously configured tables.

Reply:
Ah Ignore, It's commented out.